File: send.py
import groups
from mirai import Plain, Mirai
from appmgr import app
from user import User

async def major(msg):
    await group(groups.major, msg)


async def console(msg):
    await group(groups.console, msg)


async def test(msg):
    await group(groups.test, msg)


async def group(groupid, msg):
    await app.sendGroupMessage(groupid, msg)

async def private(uin, msg):
    if type(uin) is User:
        uin=uin.uin
    await app.sendTempMessage(groups.major, uin, msg)

File: LemonApp/Common/Send.cs
namespace Clansty.tianlang
{
    static class S
    {
        public static void Group(long group, string msg)
        {
            Robot.Send.Group(group, msg);
        }

        public static void Private(long qq, string msg)
        {
            Robot.Send.Friend(qq, msg);
        }

        public static void Major(string msg) => Group(G.major, msg);
        public static void Si(string msg) => Group(G.si, msg);
        public static void Test(string msg) => Group(G.test, msg);
    }
}